oracle-lib/README.md

```diff
@@ -25,8 +27,11 @@ project.
 
   In each case, be sure you get the version matching the versions specified
   in the `project.clj`.
```

Comment:
The markdown here doesn't seem to be formatted properly:
https://github.com/nasa/Common-Metadata-Repository/blob/b8afa03ddc5d31a729e89ef030e3ac826505ffb4/oracle-lib/README.md

Reply:
Odd. It's resolving correctly locally in my viewer. Cheers.

Reply:
Fixed.